x-pack/legacy/plugins/lens/public/editor_frame_plugin/editor_frame/suggestion_helpers.test.ts

```diff
@@ -6,11 +6,15 @@
 
 import { getSuggestions } from './suggestion_helpers';
 import { createMockVisualization } from '../mocks';
-import { TableSuggestion } from '../../types';
+import { TableSuggestion, DatasourceSuggestion } from '../../types';
 
-const generateSuggestion = (datasourceSuggestionId: number = 1, state = {}) => ({
+const generateSuggestion = (
+  datasourceSuggestionId: number = 1,
+  state = {},
+  layerId: string = 'first'
```

Comment:
Why have a default layerId here?

Reply:
because it shortens the tests below. What's your concern with that?